packages/integrations/react/server.js

```diff
@@ -101,6 +101,9 @@ async function renderToStaticMarkup(Component, props, { default: children, ...sl
 		formState,
 	};
 	let html;
+	if (opts.experimentalDisableStreaming) {
+		html = ReactDOM.renderToString(vnode);
+	}
 	if ('renderToReadableStream' in ReactDOM) {
```

Comment:
Should this be an else if?
```suggestion
	else if ('renderToReadableStream' in ReactDOM) {
```

Reply:
Oh good catch!